WordPress/Classes/Utility/WebKitViewController.swift

```diff
@@ -359,9 +359,18 @@ class WebKitViewController: UIViewController, WebKitAuthenticatable {
 
     /// Sets the width of the web preview
     /// - Parameter width: The width value to set the webView to
-    func setWidth(_ width: CGFloat?) {
+    func setWidth(_ width: CGFloat?, viewSize: CGSize? = nil) {
```

Comment:
Nit: If we only need the width from `viewSize`, maybe we should just pass that in?